### PR TITLE
metrics: add comments for metrics ranges

### PR DESCRIPTION
**Pull request by jackysp**
### What problem does this PR solve? 
It is not easy to understand the ranges of metrics, especially for the duration. And some ranges are not suitable for our users.

### What is changed and how it works?
Add comments for metrics ranges and adjust parts of them.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported variable/fields change

### CODE REVIEW

File: metrics/ddl.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ var (
 			Subsystem: "ddl",
 			Name:      "handle_job_duration_seconds",
 			Help:      "Bucketed histogram of processing time (s) of handle jobs",
-			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 20),
+			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 22), // 10ms ~ 12hours
```
Comment (zimulala) [inline metrics/ddl.go]: Do we unify the expression？Some place we use `4096s`  like line91.

Comment (jackysp) [reply metrics/ddl.go]: I think no need to do it. 4096 is also easy to understand. And the range of this value is not accurate.

File: metrics/tikvclient.go [hunk 2]
```diff
@@ -202,18 +193,16 @@ var (
...
 			Subsystem: "tikvclient",
 			Name:      "batch_wait_duration",
-			// Min bucket is [0, 1ns).
-			Buckets: prometheus.ExponentialBuckets(1, 2, 30),
-			Help:    "batch wait duration",
+			Buckets:   prometheus.ExponentialBuckets(1, 2, 30), // 1ns ~ 1s
+			Help:      "batch wait duration",
 		})
 
 	TiKVBatchClientUnavailable = prometheus.NewHistogram(
```
Comment (zimulala) [inline metrics/tikvclient.go]: This metric doesn't be registered. Please help add it.

File: metrics/ddl.go (hunk 1 above)
Comment (zimulala) [reply metrics/ddl.go]: Got it.

File: metrics/tikvclient.go (hunk 2 above)
Comment (jackysp) [reply metrics/tikvclient.go]: added

File: metrics/tikvclient.go [hunk 3]
```diff
@@ -143,7 +134,7 @@ var (
 			Subsystem: "tikvclient",
 			Name:      "rawkv_kv_size_bytes",
 			Help:      "Size of key/value to put, in bytes.",
-			Buckets:   prometheus.ExponentialBuckets(1, 2, 21),
+			Buckets:   prometheus.ExponentialBuckets(1, 2, 21), // 1Byte ~ 2MB
```
Comment (shenli) [inline metrics/tikvclient.go]: The max size of key plus value is 6MB. So I think this metric need larger bucket.

Comment (jackysp) [reply metrics/tikvclient.go]: addressed, PTAL @shenli

File: metrics/ddl.go (hunk 1 above)
Comment (zz-jason) [reply metrics/ddl.go]: could add more comments about how these ranges are calculated?

Comment (jackysp) [reply metrics/ddl.go]: For this one only or all of them?

**Pull request merged**